CMakeLists.txt

```diff
@@ -25,9 +25,8 @@ set(ENABLE_TESTS Off CACHE BOOL "Enable BLT and RAJA tests")
 set(ENABLE_EXAMPLES Off CACHE BOOL "Enable RAJA examples")
 set(RAJA_ENABLE_EXERCISES Off CACHE BOOL "Enable RAJA exercises")
 
-set(CMAKE_CXX_STANDARD 11)
-set(BLT_CXX_STANDARD 11)
-
+set(CMAKE_CXX_STANDARD 14)
```

Comment:
Does Kokkos require C++14? If so, let's make this conditional depending on `ENABLE_KOKKOS`